runtime/compiler/ilgen/Walker.cpp

```diff
@@ -6754,6 +6786,16 @@ TR_J9ByteCodeIlGenerator::genNewArray(int32_t typeIndex)
         }
      }
 
+   // Special case for handling Arrays.copyOfRangeByte when called from StringBuilder.tostring method
```

Comment:
`StringBuilder.toString` - Follow the same signature as actual method.

Reply:
Fixed